kythe/proto/metadata.proto

```diff
@@ -17,6 +17,8 @@ syntax = "proto3";
 
 package kythe.proto.metadata;
 
+option java_package = "com.google.devtools.kythe.proto";
```

Comment:
minor: this change necessitates running `kythe/proto/generate_protobufs.py` and committing the result